libdnf/dnf-rpmts.cpp

```diff
@@ -36,32 +36,64 @@
 #include <rpm/rpmlog.h>
 #include <rpm/rpmdb.h>
 
-#include "dnf-rpmts.h"
+#include "sack/packageset.hpp"
+#include "hy-package-private.hpp"
+#include "dnf-rpmts-private.hpp"
 #include "dnf-types.h"
 #include "dnf-utils.h"
 
 #include "utils/bgettext/bgettext-lib.h"
+#include "dnf-package.h"
+
+
+static gboolean
+test_fail_safe(Header * hdr, DnfPackage * pkg, GError **error)
+{
+    if (dnf_package_installed(pkg)) {
+        return TRUE;
+    }
+    if (strcmp(dnf_package_get_reponame(pkg), HY_CMDLINE_REPO_NAME) == 0) {
+        return TRUE;
+    }
+    if (auto repo = dnf_package_get_repo(pkg)) {
+        if (dnf_repo_get_module_hotfixes(repo)) {
+            return TRUE;
+        }
+    } else {
+        return TRUE;
+    }
+    rpmtd td = rpmtdNew();
+    if (headerGet(*hdr, RPMTAG_MODULARITYLABEL, td, HEADERGET_MINMEM)) {
```

Comment:
You can shorten code:
Add line `gboolean ret = TRUE;` and then removing some lines. Info below.